app/controllers/spree/admin/digitals_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Spree
+  module Admin
+    class DigitalsController < ResourceController
+      belongs_to 'spree/product', find_by: :slug
```

Comment:
@MatthewKennedy arent't digitals tied to variants?

Reply:
Hi @damianlegawiec The digitals are linked to variants through the product:

A Product with no Variants:
<img width="1440" alt="Screenshot 2021-11-15 at 10 37 25" src="https://user-images.githubusercontent.com/1240766/141767363-189de53c-bbea-4e40-bb1d-ff00ee01cf54.png">


A Product with Variants:
<img width="1440" alt="Screenshot 2021-11-15 at 10 38 17" src="https://user-images.githubusercontent.com/1240766/141767394-956211d0-25a8-43eb-8c60-6b803148f64f.png">